frontend/src/components /registers.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useRef } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { useHistory } from "react-router";
+
+// const url = "http://localhost:5000/";
+const url = "https://anna-sarah-auth-project.herokuapp.com/";
+
+export const Register = () => {
+  // hover function turn on and off
+  const inputRef = useRef();
+  const [name, setName] = useState(" ");
+  const [email, setEmail] = useState(" ");
+  const [password, setPassword] = useState(" ");
```

Comment:
This will cause `password` to start with a star in the input box, is this intended?